src/Compilers/CSharp/Test/Semantic/Semantics/Utf8StringsLiteralsTests.cs

```diff
@@ -3066,41 +2928,13 @@ Span<byte> Test()
             var node = tree.GetCompilationUnitRoot().DescendantNodes().OfType<ReturnStatementSyntax>().Single().Expression;
 
             var symbolInfo = model.GetSymbolInfo(node);
-            Assert.Equal("System.Span<System.Byte> System.Span<System.Byte>.op_Implicit(System.Byte[]? array)", symbolInfo.Symbol.ToTestDisplayString());
-
-            var typeInfo = model.GetTypeInfo(node);
-            Assert.Equal("System.Byte[]", typeInfo.Type.ToTestDisplayString());
-            Assert.Equal("System.Span<System.Byte>", typeInfo.ConvertedType.ToTestDisplayString());
-
-            Assert.True(model.GetConversion(node).IsUserDefined);
-        }
-
-        [ConditionalFact(typeof(CoreClrOnly))]
-        public void SemanticModel_10()
-        {
-            var source = @"
-using System;
-class C
-{
-    Span<byte> Test()
-    {
-        return (Span<byte>)""1""u8;
-    }
-}
-";
-            var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugDll);
-            var tree = comp.SyntaxTrees.Single();
-            var model = comp.GetSemanticModel(tree);
-            var node = tree.GetCompilationUnitRoot().DescendantNodes().OfType<ReturnStatementSyntax>().Single().Expression;
-
-            var symbolInfo = model.GetSymbolInfo(node);
-            Assert.Equal("System.Span<System.Byte> System.Span<System.Byte>.op_Implicit(System.Byte[]? array)", symbolInfo.Symbol.ToTestDisplayString());
+            Assert.Null(symbolInfo.Symbol);
```

Comment:
Consider also verifying diagnostics

Reply:
> Consider also verifying diagnostics

I think other tests provide enough coverage for diagnostics.